app/server.js

```diff
@@ -259,6 +260,8 @@ const isRobotRequest = agent =>
 
 export default function (req, res, next) {
   const config = getConfiguration(req);
+  addMetaData(config); // add dynamic metadata content
+  const application = appCreator(config);
```

Comment:
Application is already defined below https://github.com/HSLdevcom/digitransit-ui/pull/1275/files#diff-f53e59f241bb62f0261e245350683e07R279